.gitignore

```diff
@@ -11,30 +11,31 @@ build/
 .idea
 *.iml
 
-# Ignores from bower
 
-bower.json
```

Comment:
removed bower since it is no longer used. 
Otherwise refactored for readability.